### ISSUE TITLE
Redundant checks 1557, 1558, 1560, 1561, 1654, 1703, 1706 and 1707 duplicate check 2000

### ISSUE THREAD

**Issue by FrankHippmann** [iho-ohi/ENCWG-S58-ENC-Validation-Checks#53]
S-58 should aim to specify each check once only. This is important for the following reasons.

1. When an update occurs, e.g., to the allowable values of T_MTOD for T_HMON, this needs to be updated in two separate places. Inconsistencies can easily occur.
3. This makes referencing check IDs in validation software more complex and confusing. The idea of check 2000 is that it is data driven via the table of allowable values. When check 2000 is referenced as an error, the other seven check IDs need to be referenced as well, since they are instances of check 2000. We get feedback from customers that this is confusing.
4. There are additional comments in check 2000 stating that the duplications exist. Why not just remove the duplicates and remove the comments. This will simplify the document by removing the unnecessary cross-referencing of checks.  In engineering, it is best to fix the root of the problem, rather than work around it.
5. The same issue is occurring with the S-100 and S-101 checks under development. We should set a guideline that this should be avoided, and that when duplicates are found that they are removed.

**Comment by Richard-Fowle** [comment]
I agree with Frank's assessment and propose the removal of checks 1557, 1558, 1560, 1561, 1654, 1703, 1706 and 1707 that are duplicated by check 2000.

**Comment by Richard-Fowle** [comment]
Changes added to draft NE 9.0.0 of S-58
